cmd/influxd/config.go

```diff
@@ -312,6 +313,11 @@ func (d *Duration) UnmarshalText(text []byte) error {
 	return nil
 }
 
+// WriteConfigFile writes the config to the specified writer
+func WriteConfigFile(c *Config, w io.Writer) error {
```

Comment:
I would have expected this to be a method on the `Config` type. Am I missing something?